superset-frontend/webpack.config.js

```diff
@@ -199,57 +189,29 @@ const config = {
       {
         test: /\.tsx?$/,
         use: [
-          { loader: 'cache-loader' },
-          {
-            loader: 'thread-loader',
-            options: {
-              // there should be 1 cpu for the fork-ts-checker-webpack-plugin
-              workers: os.cpus().length - 1,
-            },
-          },
+          'thread-loader',
+          babelLoader,
           {
             loader: 'ts-loader',
             options: {
               // transpile only in happyPack mode
               // type checking is done via fork-ts-checker-webpack-plugin
               happyPackMode: true,
+              transpileOnly: true,
```

Comment:
`transpileOnly: true` is implied by `happyPackMode: true`. Here we are just being explicit.

Reply:
We should get rid of `ts-loader` and just use `babel-loader` for it with `@babel/preset-typescript`

Reply:
I tried that but didn't see much improvement on things, especially on building speed ([this guy seems to agree](https://blog.logrocket.com/choosing-between-babel-and-typescript-4ed1ad563e41/)). Also remembering running into some import errors, not sure if it's related to babel typescript though. Might take another dab later.